relay-server/src/utils/dynamic_sampling.rs

```diff
@@ -101,6 +101,51 @@ pub fn get_sampling_result(
     SamplingResult::determine_from_sampling_match(sampling_result)
 }
 
+pub fn is_trace_fully_sampled(
```

Comment:
Could you add a comment describing the intended outside visible behavior? This would in part pick up on the code comments written below.